FF1Lib/Hacks.cs

```diff
@@ -15,72 +15,72 @@ public partial class FF1Rom : NesRom
 		public const int PartyShuffleSize = 3;

```

Comment:
Accidentally added this file and it isn't needed, unsure how to remove it, the updates are all line endings